### PR TITLE
Create actions-jimo

### PR DESCRIPTION
**Pull request by imsamdez**
Documentation for the Jimo Browser Destination

### Proposed changes

Added an index.md in `src/connections/destinations/catalog/actions-jimo/index.md`

### Merge timing
- ASAP once approved?

### CODE REVIEW

File: src/connections/destinations/catalog/actions-jimo/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Jimo Destination
+---
+{% include content/plan-grid.md name="actions" %}
+
+[Jimo](https://usejimo.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} helps you to publish personalized product experiences to elevate adoption and user engagement without code as a layer on top of your app
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-jimo/index.md:6]:
```suggestion
[Jimo](https://usejimo.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} helps you to publish personalized product experiences to elevate adoption and user engagement without code as a layer on top of your app.
```

File: src/connections/destinations/catalog/actions-jimo/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+This destination is maintained by Jimo. For any issues with the destination, [contact their Support team](mailto:support@usejimo.com).
+
+## Getting started
+
+1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank”} search for "Jimo".
+2. Select Jimo and click **Add Destination**.
+3. Select an existing Source to connect to <Jimo (Actions).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-jimo/index.md:14]:
```suggestion
3. Select an existing Source to connect to Jimo (Actions).
```

File: src/connections/destinations/catalog/actions-jimo/index.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+4. Go to the [Jimo dashboard](https://i.usejimo.com/settings/general){:target="_blank"}, find and copy your **project id**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-jimo/index.md:15]:
```suggestion
4. Open your [Jimo dashboard](https://i.usejimo.com/settings/general){:target="_blank"}, find and copy your **project id**.
```

File: src/connections/destinations/catalog/actions-jimo/index.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+5. Enter the **project id** in the Jimo destination settings in Segment.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-jimo/index.md:16]:
```suggestion
5. Return to Segment and open the destination settings for the Jimo (Actions) destination that you just created. Enter your Jimo **project id**.
```

**Review by forstisabella — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/actions-jimo/index.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+title: Jimo Destination
+---
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-jimo/index.md:3]:
```suggestion
---
title: Jimo (Actions) Destination
id: 652d4cf5e00c0147e6eaf5e7
beta: true
---
```

File: src/connections/destinations/catalog/actions-jimo/index.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+This destination is maintained by Jimo. For any issues with the destination, [contact their Support team](mailto:support@usejimo.com).
+
+## Getting started
+
+1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank”} search for "Jimo".
+2. Select Jimo and click **Add Destination**.
+3. Select an existing Source to connect to Jimo (Actions).
+4. Open your [Jimo dashboard](https://i.usejimo.com/settings/general){:target="_blank"}, find and copy your **project id**.
+5. Return to Segment and open the destination settings for the Jimo (Actions) destination that you just created. Enter your Jimo **project id**.
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-jimo/index.md:16]:
```suggestion
1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank”} search for "Jimo (Actions)".
2. Select Jimo (Actions) and click **Add Destination**.
3. Select an existing Source to connect to Jimo (Actions).
4. Open your [Jimo dashboard](https://i.usejimo.com/settings/general){:target="_blank"}, find and copy your **project id**.
5. Return to Segment and open the destination settings for the Jimo (Actions) destination that you just created. Enter your Jimo **project id**.
```

**Review by tcgilbert — CHANGES_REQUESTED** [review]
just a couple changes, but lgtm

**Comment by imsamdez** [comment]
Committed, thanks for the suggestions!

**Review by tcgilbert — APPROVED** [review]

**Review by forstisabella — APPROVED** [review]

**Comment by tcgilbert** [comment]
@imsamdez since you have submitted your documentation. If you have tested the integration, and are certain it behaves as expected, we can make your integration live. Is that the case, or would you like to wait?

**Comment by imsamdez** [comment]
> @imsamdez since you have submitted your documentation. If you have tested the integration, and are certain it behaves as expected, we can make your integration live. Is that the case, or would you like to wait?

Hey @tcgilbert, you can make it live yes, thanks!

**Comment by tcgilbert** [comment]
@imsamdez great! this will go live tomorrow afternoon when the docs site rebuilds. I'll send an update to you then

**Pull request merged**